__tests__/__utils__/assertions.ts

```diff
@@ -61,6 +62,7 @@ export class Client {
         cache: global.cache,
         swrQueue: emptySwrQueue,
         authServices,
+        timer: this.context?.timer ?? createTimer(),
```

Comment:
```suggestion
        timer: global.timer,
```

This is the error, here you need to give the mock timer to the test (here the context for the tests is created)